episodes/00-repo-access.md

```diff
@@ -46,6 +46,20 @@ as a collaborator by using the forking model.
 [^permission-personal]: The [GitHub documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/permission-levels-for-a-personal-account-repository) has more information on
 permissions for a repository on a personal account.
 
+#### Checking Access
+
+As a collaborator you will be able to edit files on GitHub.
+
+1. Go to the GitHub repository.
+2. Scroll down to the top of the rendered README.
+3. In the top right corner of the README click on the **Edit file** pencil icon.
+
+If you aren't a collaborator you will see the message:
+
+> Sorry, you're not able to edit this repository directly
```

Comment:
Maybe a nitpick, but the message

> You need to fork this repository to propose changes.

is more prominent.

Also, if they have created a fork at some point in the past they will see the editor with a message at the top:

> You’re making changes in a project you don’t have write access to...